package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf static/dist",
     "nodecheckversion": "node ./scripts/node-checkversion.js",
     "preinstall": "npm run nodecheckversion",
-    "postinstall": "sh ./scripts/upstream.sh && npm run build",
```

Comment:
This only sets the upstream repo. Why remove? If this stays removed, please make a note for users on how to set their upstream.